astro/src/content/docs/lifecycle/manage-users/account-management/_account-logout.mdx

```diff
@@ -0,0 +1,7 @@
+If you have enabled the [self-service account management](/docs/lifecycle/manage-users/account-management/) functionality of FusionAuth, this has a separate session from the FusionAuth SSO session or your application session.
+
+You must use the account logout endpoint to kill this session.
```

Comment:
Not sure about the use of "kill." What about "end" or "terminate"?

Reply:
Good catch.